ext/net/lib.deno_net.d.ts

```diff
@@ -110,6 +113,9 @@ declare namespace Deno {
     keyFile?: string;
 
     transport?: "tcp";
+    /** SO_REUSEADDR option for socket.
+     * If not specified, defaults to true in Linux, false in Windows. */
+    reuseAddress?: boolean;
```

Comment:
This doesn't need to be added to ListenTlsOptions because it gets inherited from ListenOptions.

Reply:
Unnecessary field and docstrings removed